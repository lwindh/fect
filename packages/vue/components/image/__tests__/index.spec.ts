import { mount } from '@vue/test-utils'
import Image from '../index'
import { later } from '../../../tests'

describe('Image', () => {
  it('should be render as element', () => {
    const img = mount(Image)
    expect(img.html()).toMatchSnapshot()
    expect(() => img.unmount()).not.toThrow()
  })
  it('should  support skeleton', () => {
    const wrapper = mount(Image, {
      props: {
        skeleton: true,
        height: '240px',
        width: '540px',
        src: '1'
      }
    })
    expect(wrapper.find('.skeleton')).toBeTruthy()
  })
})
