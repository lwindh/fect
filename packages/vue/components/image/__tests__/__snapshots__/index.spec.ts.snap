// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should be render as element 1`] = `
<div class="fect-image" style="width: auto; height: auto;">
  <!----><img src="" width="0" height="0" style="display: block;">
</div>
`;
